src/index.ts

```diff
@@ -160,7 +160,7 @@ export class WafPlugin {
               },
               Statement: {
                 GeoMatchStatement: {
-                  CountryCodes: ["GU", "PR", "US", "UM", "VI", "MP"],
+                  CountryCodes: ["AS","GU", "PR", "US", "UM", "VI", "MP"],
```

Comment:
@dwhitestratiform Should we add in the 3 Compact of Free Association countries too? It looks like they have full access to Medicaid benefits as of Dec 2020 to the appropriations act from that year.

```suggestion
                  CountryCodes: ["AS", "FM", "GU", "MH", "MP", "PR", "PW", "UM", "US", "VI"],
```

That would give us: 

`Territories`
- American Samoa (AS)
- Guam (GU)
- Northern Mariana Islands (MP)
- Puerto Rico (PR)
- US Virgin Islands (VI)
- Puerto Rico (PR)

`Compacts of Free Association Countries`
- Federated States of Micronesia (FM)
- Marshall Islands (MH)
- Palau (PW)